Nesteo.Server/Services/IInspectionService.cs

```diff
@@ -10,5 +11,7 @@ public interface IInspectionService: ICrudService<Inspection, int>
         IAsyncEnumerable<InspectionPreview> GetAllPreviewsAsync();
 
         Task<InspectionPreview> FindPreviewByIdAsync(int id, CancellationToken cancellationToken = default);
+
+        IAsyncEnumerable<Inspection> FindByBoxIdAsync(string id);
```

Comment:
Please name it `GetAllForNestingBoxIdAsync` instead.
I prefer to use "For" when the method only returns items that are **for** another given item (like a filter) and "By" when I select one single item **by** some key.

Also naming the method parameter `nestingBoxId` here would make more clear that this is about the ID of the nesting box, even though it's inside the inspection service.